pages/electronics-library.tsx

```diff
@@ -1,12 +1,14 @@
 /* eslint-disable @next/next/no-img-element */
 import { Searcher } from 'fast-fuzzy';
-
+import Link from 'next/link'
+import { useRouter } from 'next/router'
 import { GuideLink } from '../components/GuideLink';
 import { useState, useEffect, useRef } from 'react';
 import Modal from '../components/Modal';
 import { google } from 'googleapis';
 import styles from '../styles/Page.module.scss';
 import cx from 'classnames';
+import { constants } from 'buffer';
```

Comment:
what is this for?

Reply:
Ah an artifact from being a VSCode novice... I thought I typed in const with a typo but I think it autocompleted to constant and then it added the import. I removed this.